src/cloudant/_messages.py

```diff
@@ -16,9 +16,61 @@
 Module that contains exception messages for the Cloudant Python client
 library.
 """
+ARGUMENT_ERROR = {
+    100: 'A general Cloudant argument error was raised.',
+    # Client
+    101: 'Invalid year and/or month supplied.  Found: year - {0}, month - {1}',
+    # Database
+    102: 'Invalid role(s) provided: {0}.  Valid roles are: {1}.',
+    103: 'Invalid index type: {0}.  '
+         'Index type must be either \"json\" or \"text\"',
+    104: 'A single query/q parameter is required.  Found: {0}',
+    105: 'Invalid argument: {0}',
+    106: 'Argument {0} is not an instance of expected type: {1}',
+    # Design document
+    107: 'View {0} already exists in this design doc',
+    108: 'An index with name {0} already exists in this design doc.',
+    109: 'A list with name {0} already exists in this design doc.',
+    110: 'A show function with name {0} already exists in this design doc.',
+    111: 'View {0} does not exist in this design doc.',
+    112: 'An index with name {0} does not exist in this design doc.',
+    113: 'A list with name {0} does not exist in this design doc.',
+    114: 'A show function with name {0} does not exist in this design doc.',
+    # Feed
+    115: 'Error converting argument {0}: {1}',
+    116: 'Invalid argument {0}',  # py_to_couch_validate
+    117: 'Argument {0} not instance of expected type: {1}',
+    118: 'Argument {0} must be > 0. Found: {1}',
+    119: 'Invalid value ({0}) for feed option.  Must be one of {1}',
+    120: 'Invalid value ({0}) for style option.  Must be main_only, '
+         'or all_docs.',
+    121: 'Invalid infinite feed option: {0}.  Must be set to continuous.',
+    # Index
+    122: 'The design document id: {0} is not a string.',
+    123: 'The index name: {0} is not a string.',
+    124: '{0} provided as argument(s).  A JSON index requires that '
+         'only a \'fields\' argument is provided.',
+    125: 'Deleting an index requires a design document id be provided.',
+    126: 'Deleting an index requires an index name be provided.',
+    127: 'Invalid argument: {0}',
+    128: 'Argument {0} is not an instance of expected type: {1}',
+    # Query
+    129: 'Invalid argument: {0}',
+    130: 'Argument {0} is not an instance of expected type: {1}',
+    131: 'No selector in the query or the selector was empty.  '
+         'Add a selector to define the query and retry.',
+    # View
+    132: 'The map property must be a dictionary',
+    133: 'The reduce property must be a string.',
```

Comment:
Now they're grouped together, it's more obvious that the punctuation of some of these messages is a bit inconsistent. Some end in a period and some don't.